lib/buildConnectorsJson/generateSchemaFromJs.js

```diff
@@ -5,129 +5,106 @@
 var _            = require('lodash');
 var prettyTitle  = require('./prettyTitle');
 
-
-module.exports = function (config) {
-	return generateSchema(config, true);
+//Standard supported keys
+var supportedKeys = [
+	'type',
+	'title',
+	'enum',
+	'oneOf',
+	'name',
+	'default',
+	'description',
+	'lookup',
+	'format'
+];
+
+module.exports = function (rawSchema) {
+	var schema = generateSchema(rawSchema, true);
+	//Since $schema is only required at root level, assign it here.
+	schema['$schema'] = "http://json-schema.org/draft-04/schema#";
+	 // Standard restriction: top level schema never allows for additional properties
+	schema['additionalProperties'] = false;
```

Comment:
JS standard object notation should be used where possible, i.e. `schema.additionalProperties`